rocket-chat-android-widgets/src/main/res/layout/message_inline_attachment.xml

```diff
@@ -93,14 +96,12 @@
 
         </LinearLayout>
 
-        <ImageView
+        <com.facebook.drawee.view.SimpleDraweeView
                 android:id="@+id/image"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:maxHeight="200dp"
+                android:layout_width="match_parent"
+                android:layout_height="200dp"
```

Comment:
![slice](https://cloud.githubusercontent.com/assets/11763113/22378141/ed124172-e4f7-11e6-9e2d-ebc79c887528.png)

It seems that the boundary of each attachment image is larger than expected for its size.
Is it possible to solve with setting maxHeight=200 and height=wrap_content for SimpleDraweeView?